### DISCUSSION TITLE
Deploying SeleniumBase on ubuntu

### DISCUSSION THREAD

**Discussion by mubaraklightforth** [seleniumbase/SeleniumBase#3322] [Q&A] [answered]

**Comment by mdmintz** [comment, marked as answer]
Tutorial Video 3 (https://www.youtube.com/watch?v=-EpZlhGWo9k) covered that on Linux you may need to use `uc_gui_click_captcha()` instead of `uc_gui_handle_captcha()` to avoid detection.
Tutorial Video 4 (https://www.youtube.com/watch?v=Mr90iQmNsKM) covered that AWS has a non-residential IP address range, (which makes your bot detectable), and provides a solution for getting past that. Video 4 also introduces the new [CDP Mode](https://github.com/seleniumbase/SeleniumBase/blob/master/examples/cdp_mode/ReadMe.md), which is the successor to regular UC Mode.

**Reply by mubaraklightforth** [reply]
Thanks alot @mdmintz , checking it out now
